client/src/components/QuestionsPage/Details.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
```

Comment:
You can just import this component from the Item folder rather than make a new file for it.

Reply:
There are slight changes made to the file which is why I do not import that.  I'll place it into the shared folder since 2 of my pages use it